system_tests/system_test.go

```diff
@@ -586,13 +591,17 @@ CONSOLE_LOG=new`
 			By("STOMP")
 			publishAndConsumeSTOMPMsg(hostname, rabbitmqNodePort(ctx, clientSet, cluster, "stomp"), username, password, nil)
 
-			// github.com/go-stomp/stomp does not support STOMP-over-WebSockets
-
-			By("stream")
-			if strings.Contains(cluster.Spec.Image, ":3.8") || strings.HasSuffix(cluster.Spec.Image, "tanzu-rabbitmq:1") {
+			By("Streams")
+			if !hasFeatureEnabled(cluster, "stream_queue") {
```

Comment:
That's a much better approach to check whether the cluster supports streams! Thanks @MarcialRosales!